### DISCUSSION TITLE
Braden Sidoti - Clerk.dev

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#224] [Announcements]
- https://www.linkedin.com/in/braden-sidoti-25676537/
- https://twitter.com/bsinthewild?lang=en
- https://clerk.com/

## Questions

- [x] What is clerk?
- [x] What was the original problem that led you to building clerk?
- [x] Whats the big differentiator between clerk and it's competitors?
- [x] Auth is hard, what has been the hardest thing to get right at scale?
- [x] One tough thing with auth is managing and connecting multiple accounts. Could clerk help me with this sort of problem?
- [x] What are some of the cooler features that clerk makes easy?
- [x] You guys have been going for awhile and you charts recently hockey sticked. Congrats! Achieving scale is hard, what steps did you take to make sure that scale was possible? 
- [x] Pricing is always a hot topic. Especially for services people feel you should "build yourself". How does clerk pricing work and do you think it's worth it for indies and bigcos alike?
- [x] With auth being at the core of a business it could be risky to tie your company to another. Does clerk make it easy to switch off clerk?
- [x] What's next for clerk?

### Future

~- [ ] Have you seen passwordless auth? What do you think?~
- [x] What's your spiciest dev take?
- [ ] What do you think the future of building apps looks like? Will we build out the boring parts of our businesses through services?

**Comment by hipstersmoothie** [comment]
https://rivet.ironcladapp.com/
https://jsx.email/

**Comment by braden-clerk** [comment]
https://resend.com/

**Comment by just-be-dev** [comment]
Tooltips:
- https://github.com/0xType/0xProto
- https://github.com/seflless/skewed by Francois Laberge
